src/pages/Filter/Filter.scss

```
@@ -0,0 +1,302 @@
+$color-main: #fb7d27;
+$color-white: #ececec;
+$color-grey: #9b9b9b;
+
+div.UpperBox {
+  display: flex;
+  flex-direction: column;
+  border: solid;
+  border-width: thin;
+  border-radius: 10px;
+  margin: 10%;
+  margin-left: 18%;
+  margin-right: 18%;
+  border-color: $color-grey;
+
+  div.searchFilter {
+    div.selectHead {
+      display: flex;
+      justify-content: space-between;
+
+      span.title {
+        margin-left: 20px;
+        margin-top: 10px;
+        padding: 10px;
+      }
+
+      span.multipleChoice {
+        margin-right: 15px;
+        margin-top: 10px;
+        padding: 10px;
+      }
+    }
+
+    div.selectSection {
+      display: flex;
+      justify-content: center;
+      text-align: center;
+      border-top: solid;
+      border-bottom: solid;
+      border-width: thin;
+      border-color: $color-grey;
+
+      .rating {
+        display: inline-block;
+        width: 200px;
+        height: 45px;
+        margin: 20px;
+        margin-left: 20px;
+        padding: 10px;
+        border-radius: 50px;
+        border: solid;
+        border-width: thin;
+        border-color: $color-grey;
+      }
+
+      .popularity {
+        display: inline-block;
+        width: 200px;
+        height: 45px;
+        margin: 20px;
+        border-radius: 50px;
+        padding: 10px;
+        border: solid;
+        border-width: thin;
+        border-color: $color-grey;
+      }
+    }
+  }
+
+  div.priceForOnePerson {
+    border-bottom: solid;
+    border-color: $color-grey;
+    border-width: thin;
+
+    div.selectHead {
+      display: flex;
+      justify-content: space-between;
+
+      span.title {
+        margin-left: 20px;
+        margin-top: 10px;
+      }
+
+      span.multipleChoice {
+        margin-right: 20px;
+        margin-top: 10px;
+      }
+    }
+
+    div.priceSelectSection {
+      display: flex;
+      justify-content: space-around;
+      text-align: center;
+
+      span.lessThan10000 {
+        display: inline-block;
+        width: 200px;
+        height: 45px;
+        margin: 20px;
+        border-radius: 50px;
+        padding: 10px;
+        border: solid;
+        border-width: thin;
+        border-color: $color-grey;
+      }
+
+      .over10000 {
+        display: inline-block;
+        margin: 20px;
+        width: 200px;
+        height: 45px;
+        border-radius: 50px;
+        padding: 10px;
+        border: solid;
+        border-width: thin;
+        border-color: $color-grey;
+      }
+
+      .over20000 {
+        display: inline-block;
+        width: 200px;
+        height: 45px;
+        margin: 20px;
+        border-radius: 100px;
+        padding: 10px;
```

```suggestion
      .priceSelect {
        display: inline-block;
        width: 200px;
        height: 45px;
        margin: 20px;
        border-radius: 50px;
        padding: 10px;
        border: solid;
        border-width: thin;
        border-color: $color-grey;
      }
```
- 동일한 스타일에 다른 셀렉터 이름을 지정해서 중복 코드를 발생시키지 않아도 될 것 같습니다

---

수정했습니다